setup.py

```diff
@@ -32,12 +33,7 @@
     long_description=readme,
     long_description_content_type="text/markdown",
     keywords='corva, sdk',
-    packages=setuptools.find_packages(exclude=[".*", "data", "test"]),
-    install_requires=[
-        "pydantic >= 1.7.2",
-        "redis >= 3.5.3",
-        "requests >= 2.25.0",
-        "urllib3 >= 1.26.2"
-    ],
+    packages=["corva"],
+    install_requires=["pydantic >= 1.7.3", "redis >= 3.5.3", "requests >= 2.25.0"],
```

Comment:
I thought this was easier to read with each requirement on its own line as well

Reply:
fixed